RESTserver/src/main/java/com/softserve/if072/restservice/controller/GoShoppingController.java

```diff
@@ -67,4 +68,16 @@ public GoShoppingController(GoShoppingService goShoppingService) {
             return null;
         }
     }
+
+    @PostMapping("/cart")
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.OK)
+    public void getProducts(@RequestBody FormForCart carts, HttpServletResponse response) {
+        try {
+            goShoppingService.insertCart(carts);
+            LOGGER.info(String.format("Cart of user id %d was updated", carts.getUserId()));
+        } catch (Exception e) {
```

Comment:
if I try insert existing product, some not declared sql exception is thrown. When in cartDAO method, that can check it, appear, I will check it